tests/integration/local/start_api/test_start_api.py

```diff
@@ -34,8 +34,12 @@ class TestServiceHTTP10(StartApiIntegBaseClass):
 
     def setUp(self):
         self.url = "http://127.0.0.1:{}".format(self.port)
+        self.current_svn_str = HTTPConnection._http_vsn_str
         HTTPConnection._http_vsn_str = "HTTP/1.0"
```

Comment:
This line cause that the S3 `PutObject` API call fail, and so some of the test cases were failing with the following error, if they got executed after the test cases defined in this class.
```
botocore.exceptions.ClientError: An error occurred (RequestTimeout) when calling the PutObject operation (reached max retries: 4): Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.
```